go.mod

```diff
@@ -2,8 +2,11 @@ module github.com/ava-labs/avalanche-cli
 
 go 1.18
 
+replace github.com/ava-labs/avalanchego => ../avalanchego-internal/
```

Comment:
I assume it is obvious to you that this is a draft only PR because of this

Reply:
I was relying in that this is not passing CI. But will also mark it as draft